Makefile

```diff
@@ -125,42 +137,40 @@ FATFS_SRC       = $(notdir $(wildcard $(FATFS_DIR)/*.c))
 
 CSOURCES        := $(shell find $(SRC_DIR) -name '*.c')
 
-
 ifeq ($(TARGET),$(filter $(TARGET),$(F3_TARGETS)))
 # F3 TARGETS
 
 STDPERIPH_DIR   = $(ROOT)/lib/main/STM32F30x_StdPeriph_Driver
 STDPERIPH_SRC   = $(notdir $(wildcard $(STDPERIPH_DIR)/src/*.c))
 EXCLUDES        = stm32f30x_crc.c \
-                  stm32f30x_can.c
+				  stm32f30x_can.c
```

Comment:
`make` allows spaces here - in fact I had changed the `Makefile` so that all the settings of variables used spaces.

Where Makefile requires tabs is for commands.

So
```
REVO_SRC = \
            $(STM32F4xx_COMMON_SRC) \
...
CFLAGS      += $(ARCH_FLAGS) \
              $(LTO_FLAGS) \
```
etc, can use tabs, whereas
```
$(TARGET_BIN): $(TARGET_ELF)
	$(OBJCOPY) -O binary $< $@
...
clean:
	rm -f $(CLEAN_ARTIFACTS)
```
etc requires tabs

Everything above the 
```
# No user-serviceable parts below
```
line can use spaces.

I'd certainly prefer it if you changed to use 4 spaces rather than tabs above the that line and in the `.mk` files